### PR TITLE
logger

### PR DESCRIPTION
**Pull request by julia-shvabovich**

### CODE REVIEW

File: src/main/java/cinema/Main.java [hunk 1]
```diff
@@ -15,26 +15,27 @@
...
 
     public static void main(String[] args) {
         MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
-        movieService.getAll().forEach(System.out::println);
         Movie movie = new Movie();
         movie.setTitle("Fast and Furious");
         movie.setDescription("I never heard of the film");
         movieService.add(movie);
-        movieService.getAll().forEach(System.out::println);
+        movieService.getAll().forEach(LOGGER::debug);
```
Comment (andromash) [inline src/main/java/cinema/Main.java]: Better use info, not debug

**Review by andromash — COMMENTED** [review]
You need to move logger to the dao layer

File: src/main/java/cinema/Main.java (hunk 1 above)
Comment (julia-shvabovich) [reply src/main/java/cinema/Main.java]: why?

Comment (dmytro-hryhoruk) [reply src/main/java/cinema/Main.java]: with debug level we log in those information which we want to see if something has happened ( some object state from which depend the program running e.g)

File: src/main/java/cinema/Main.java [hunk 2]
```diff
@@ -15,26 +15,27 @@
 import cinema.service.ShoppingCartService;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import org.apache.log4j.Logger;
 
 public class Main {
+    private static final Logger LOGGER = Logger.getLogger(Main.class);
```
Comment (dmytro-hryhoruk) [inline src/main/java/cinema/Main.java]:
```suggestion
    private static final Logger logger = Logger.getLogger(Main.class);
```

File: src/main/java/cinema/service/impl/OrderServiceImpl.java [hunk 3]
```diff
@@ -11,9 +11,11 @@
 import java.time.LocalDateTime;
 import java.util.ArrayList;
 import java.util.List;
+import org.apache.log4j.Logger;
 
 @Service
 public class OrderServiceImpl implements OrderService {
+    private static final Logger LOGGER = Logger.getLogger(OrderServiceImpl.class);
```
Comment (dmytro-hryhoruk) [inline src/main/java/cinema/service/impl/OrderServiceImpl.java]: same

**Review by dmytro-hryhoruk — COMMENTED** [review]
apply comment to all dao layers

File: src/main/java/cinema/service/impl/OrderServiceImpl.java (hunk 3 above)
Comment (julia-shvabovich) [reply src/main/java/cinema/service/impl/OrderServiceImpl.java]: done

File: src/main/java/cinema/security/AuthenticationServiceImpl.java [hunk 4]
```diff
@@ -8,9 +8,11 @@
 import cinema.service.UserService;
 import cinema.util.HashUtil;
 import java.util.Optional;
+import org.apache.log4j.Logger;
 
 @Service
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = Logger.getLogger(AuthenticationServiceImpl.class);
```
Comment (dmytro-hryhoruk) [inline src/main/java/cinema/security/AuthenticationServiceImpl.java]: let's use logger on dao layer(all of them)

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**